.github/workflows/test-and-demo.yaml

```diff
@@ -5,7 +5,11 @@
 #
 # For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
 
-on: push
+on:
+  push:
+    branches:
+      - master
+  pull_request:
```

Comment:
I'm not entirely sure yet, but I suspect this has also somehow effectively broken the Netlify link that is commented into the PRs. I suspect that now `github.ref` in `--alias $(echo '${{ github.ref }}' | shasum | awk '{print $1}')` in the deploy step below refers no longer to the branch name but to something like `refs/pull/439/merge`.

I haven't looked closer into how to fix this yet. Might do so tomorrow.